src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -328,6 +328,123 @@ public void testTenantPermissionsNullPermList(TestContext context) {
     context.assertEquals(201, response.code);
   }
 
+  // Test that a permission can be used in another module if first one is deleted MODPERMS-130
```

Comment:
Maybe add to this comment that one situation where this might occur is when two modules implement the same interface.

Reply:
Sure. I've added something.